tests/integration/high_availability/test_upgrade.py

```diff
@@ -71,7 +71,7 @@ async def test_upgrade_charms(ops_test: OpsTest, continuous_writes) -> None:
 
     # 8.0.33
```

Comment:
8.0.34?

Reply:
addressed in 351c72ce2ca53ed63bdec3dedcca950260e2e7db